minitask1/Lewis/TurtlebotDriving.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+import rospy
+import tf
+
+from geometry_msgs.msg import Twist
+from math import radians
+
+# Drives the Turtlebot in a 1m sqaure using open-loop control.
+class TurtlebotDriving():
+    
+    def __init__(self):
+        self.pub = rospy.Publisher('cmd_vel', Twist, queue_size = 10)
+        rospy.init_node('TurtlebotDriving', anonymous = True)
+
+        # Forward message to move the Turtlebot forward at 0.1m/s
+        forwardMessage = Twist()
+        forwardMessage.linear.x = 0.1
+        forwardMessage.angular.z = 0
+
+        # Left message to turn the Turtlebot by 10 degrees.
+        leftMessage = Twist()
+        leftMessage.linear.x = 0
+        leftMessage.angular.z = radians(10)
+
+        # Stop message to stop the Turtlebot
+        stopMessage = Twist()
+        stopMessage.linear.x = 0
+        stopMessage.angular.z = 0
+
+        # Variables to keep track of square drawing progress
+        sideCount = 0
+        turnCount = 0
+
+        while not rospy.is_shutdown() and sideCount != 4 and turnCount != 4:
```

Comment:
I think a condition like `sideCount < 4` is a lot clearer than `sideCount != 4` as we know that sidecount should be increasing. In this case it doesn't make any functional difference but I think its clearer and simpler